### DISCUSSION TITLE
Should server-side software of Dexie for sync'ing data be open-source too

### DISCUSSION THREAD

**Discussion by datdinhquoc** [dexie/Dexie.js#1311] [Q&A] [answered]
Dexie is opening a cloud for sync'ing from Dexie.js, where's the project for the server-side software? is it open-source too? I wish to run it on my own servers.

**Comment by dheimoz** [comment, marked as answer]
Hi @datdinhquoc 
Take a look at
https://dexie.org/cloud/pricing
